docs/Shaders/Vignette.md

```diff
@@ -0,0 +1,100 @@
+# Vignette
+
+The Vignette shader allows you to apply a vignette effect to a texture.
+
+## Usage
+
+If you want to use shaders that come with the Lightning framework you need to import Lightning first.
+
+```js
+import Lightning from '@lightningjs/core'
+```
+
+### Initialize
+
+To use the Vignette shader you need to apply it to the shader property of an element:
+
+```js
+//in template
+class MyApp extends Lightning.Application {
+    static _template() {
+        return {
+            Picture: {w: 300, h: 300, src: 'MyImage.jpg', shader: {type: Lightning.shaders.Vignette, magnitude: 3, intensity: 0.3}}
+        }
+    }
+}
+
+//in a component
+this.tag('Vignette').shader = {type: Lightning.shaders.Vignette, magnitude: 3, intensity: 0.3}
+```
+
+You can customize the appearance of the Vignette shader with the following setters.
+
+
+## Setters
+
+### magnitude
+You can set the magnitude of the Vignette shader with a `float`.
```

Comment:
Explain a bit better what is meant by `magnitude`?

Reply:
changed to:

You can set the magnitude of the Vignette shader with a `float`. The value of magnitude determines the degrees of fading around the light source.